src/registry/selectboxes/helpers.ts

```diff
@@ -0,0 +1,10 @@
+import {SelectboxesComponentSchema} from '@open-formulieren/types';
+import {Option} from '@open-formulieren/types/lib/formio/common';
+
+// A type guard is needed because TS cannot figure out it's a discriminated union
+// when the discriminator is nested.
```

Comment:
Maybe reference the TS issue so that we can track it more easily? https://github.com/microsoft/TypeScript/issues/18758

Reply:
Oh I didn't actually find the TS issue but just concluded this from various TS playground experiments :P I will add it